docker/SERVER/docker-compose.yaml

```diff
@@ -0,0 +1,63 @@
+version: '3.8'
```

Comment:
Why was this file named .yaml and old deleted?

Reply:
I've replaced this whole file with a known working example I commonly use on my servers.

I can rename to yml as per the original, but basically it's a new file.

Reply:
It's functionally the same file, even though all lines have changed. Better to keep the same extension as before =)

Reply:
Ok